web/src/popover_menus.js

```diff
@@ -243,7 +244,9 @@ export function toggle_popover_menu(target, popover_props, options) {
     const instance = target._tippy;
     let mobile_popover_props = {};
 
-    if (options?.show_as_overlay) {
+    // If the window is mobile-sized, we will render the
+    // emoji popover centered on the screen with the overlay.
+    if (options?.show_as_overlay && window.innerWidth <= media_breakpoints_num.md) {
```

Comment:
I am not sure about this change -- I think it makes sense for individual popovers to be responsible for deciding based on the shape of the popover itself when it wants to be placed here. E.g. I can imagine a popover that just always wants to be centered. perhaps.

The ideal API might be one where we have `popper` options decide if there's space to pop something normally or not, but I don't think that's possible.